specifications/api/swagger.yaml

```diff
@@ -1908,19 +2401,12 @@ x-responsesErr404: &responsesErr404
       application/json:
         schema: *responsesErr
 
-x-responsesErr405: &responsesErr405
```

Comment:
Do we think it is irrelevant to highlight in API doc? Basically every endpoint may return this...

Reply:
I don't think so. The spec includes which method to use. If you follow the spec, 405 will never be returned.

Reply:
Then `406` and `415` should be removed also I guess, no?

Reply:
415 maybe, but the swagger spec doesn't document enough for 406 to not be a concern afais.